src/libraries/System.Private.Xml/src/System/Xml/Xslt/XsltArgumentList.cs

```diff
@@ -10,24 +11,24 @@ public abstract class XsltMessageEncounteredEventArgs : EventArgs
         public abstract string Message { get; }
     }
 
-    public delegate void XsltMessageEncounteredEventHandler(object sender, XsltMessageEncounteredEventArgs e);
+    public delegate void XsltMessageEncounteredEventHandler(object? sender, XsltMessageEncounteredEventArgs e);
```

Comment:
Why is `sender` nullable?

Reply:
good point, I marked it as nullable with being more permissive in mind but in case of delegates it actually puts more burden on users.. will change to non-null